src/site/layouts/health_care_region_health_services_page.drupal.liquid

```diff
@@ -116,7 +116,7 @@
                 {% if featuredContentHealthServices.length %}
                     <section class="featured-services" id="featured-services">
                         <h3>In the spotlight at {{ regionOrOffice | remove: "health care" }}</h3>
-                        <ul class="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4">
+                        <ul class="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row">
```

Comment:
Out of curiosity, how is this working out for you? Do you like using a bunch of `vads-` classes like this?

To be clear, this is the recommended approach, I'm just trying to get a feel for developer satisfaction.

Reply:
it is ok to me, especially for what it solves. 
yes, it can get kind of wordy and long lines of css classes, but i think the benefits of having a system like this is better than most.

Reply:
Good to hear! Thanks for your feedback! :raised_hands: